README.md

```diff
@@ -1 +1,228 @@
-# capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  
+
+  <h3><b>Capstone project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [desktop version] <a name="about-project"></a>
+
+- Open  repository
+- make a feature branch
+- in the feature branch this where i started the project
+- setup the the github workflow 
+- used html file for the header
+- used css file to set a color to the header 
+
+**[Portfolio]** is a...
+
+## 🛠html and css<a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+- Open  repository
+- make a feature branch
+- in the feature branch this where i started the project
+- setup the the github workflow 
+- used html file for the header
+- used css file to set a color to the header
+
+To get a local copy up and running, follow these steps.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **[made homepage]**
+- **[made aboutpage]**
+- **[made the mobile menu]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://lethabomot.github.io/capstone-project/)
+- [video link](https://www.loom.com/share/b76c214b9e4547ecbdc9a09b8ff479b1)
```

Comment:
- [ ] As requested by the previous reviewer kindly consider adding a highlight about a piece of the code you finds interesting or you are proud of making.